src/venue/service.js

```diff
@@ -73,11 +73,22 @@ angular
 
         options.start = options.start || moment().startOf('month').format("YYYY-MM-DD");
         options.end = options.end || moment().endOf('month').format("YYYY-MM-DD");
-        options.ticket = options.ticket || 'false';
 
         return Venue.listInventory(options).$promise;
       };
 
+      VenueService.prototype.listActiveInventory = function(venueId, options) {
+        
+        if (!venueId) throw new Error('venueId is required');
+        if (!options) throw new Error('options.required');
+
+        options.id = venueId;
+        options.start = options.start || moment().startOf('month').unix();
```

Comment:
Good point. I wasn't quite sure what to default if the start/end values weren't provided, so I just used the defaults from ```VenueService.listInventory```. I've update ```VenueService.listActiveInventory``` such that ```options.start``` and ```options.end`` are now required, and no longer have default values.